test/export/test_experimental.py

```diff
@@ -439,6 +439,95 @@ def make_inputs():
         test_inputs = make_inputs()
         self.assertEqual(gm(*test_inputs), foo(*test_inputs))
 
+    def test_dynamo_graph_capture_closure(self):
+        from torch.export import Dim
+
+        N = 3
+
+        class MyModel(torch.nn.Module):
```

Comment:
Can you add a closed over tensor? It seems you only have an integer here.

Reply:
Yeah good point. Added